src/main/java/dinkplugin/util/ColorAdapter.java

```diff
@@ -0,0 +1,23 @@
+package dinkplugin.util;
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+import java.awt.Color;
+import java.io.IOException;
+
+public class ColorAdapter extends TypeAdapter<Color> {
```

Comment:
Can we add a comment here explaining why we need a color adapter? I'm assuming it's a json serialize/deserialize class for the `java.awt.Color` type?